CityOfDetroit/bloom
# 1322/section 8 filtering

## Issue

- Closes #1321 

## Description

This PR adds a new field to the listing filter on the public side which allows users to filter by listings that have specified that they accept section 8 housing choice vouchers. It also includes translations for the filter field name.

## How Can This Be Tested/Reviewed?

This can be tested on the Detroit Public site by clicking 'See All Rentals', clicking filter, and checking the box that says 'Accepts Section 8 Housing Choice Vouchers. After applying this filter, you should see the reduced  selection of listings that meet this criteria.. 

Note: In order to unblock this PR's development, I merged in the section-8-partners-update branch so that I could utilize the new field.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
